### ISSUE TITLE
issue about the conv2d operation

### ISSUE THREAD

**Issue by kelvinleen** [guoyinwang/LEAM#20]
as in https://github.com/guoyinwang/LEAM/issues/6#issue-347848059, the implementation is not same as what the paper shows in equation 3. 

i think line50 in model.py should be replaced as follows:
    G = tf.expand_dims(G, axis=-1) # b*s*c*1  filter: ngram*1 filter_num:1 
    Att_v = tf.contrib.layers.conv2d(G, num_outputs=1, kernel_size=[opt.ngram,1], padding='SAME',activation_fn=tf.nn.relu) #b * s *  c * 1
    Att_v = tf.squeeze(Att_v,)

**Comment by SCAUapc** [comment]
I have the same problem as you. And i agree with you. But I wish can get the reply by the author

**Comment by AlexMoreo** [comment]
I think you are right. I had to do exactly the same modification (If only had I see your comment before... :D)
